requirements.txt

```diff
@@ -2,3 +2,5 @@ matplotlib
 farneback3d
 pycuda==2020.1
 #scikit-image
+mrcfile
+biopython
```

Comment:
Biopython is already a requirement of scipion-em https://github.com/scipion-em/scipion-em/blob/devel/requirements.txt#L2, we better keep this list as short as possible